sfs/mono/wfs.py

```diff
@@ -155,8 +155,101 @@ def _point(omega, x0, n0, xs, c=None):
 
 
 def point_25d(omega, x0, n0, xs, xref=[0, 0, 0], c=None, omalias=None):
+    r"""Driving function for 2.5-dimensional WFS of a virtual point source.
+
+    .. versionchanged:: 0.5
+        see notes, old handling of `point_25d()` is now `point_25d_legacy()`
+
+    Parameters
+    ----------
+    omega : float
+        Angular frequency of point source.
+    x0 : (N, 3) array_like
+        Sequence of secondary source positions.
+    n0 : (N, 3) array_like
+        Sequence of normal vectors of secondary sources.
+    xs : (3,) array_like
+        Position of virtual point source.
+    xref : (3,) array_like, optional
+        Reference point xref or contour xref(x0) for amplitude correct
+        synthesis.
+    c : float, optional
+        Speed of sound in m/s.
+    omalias: float, optional
+        Angular frequency where spatial aliasing becomes prominent.
+
+    Returns
+    -------
+    d : (N,) numpy.ndarray
+        Complex weights of secondary sources.
+    selection : (N,) numpy.ndarray
+        Boolean array containing ``True`` or ``False`` depending on
+        whether the corresponding secondary source is "active" or not.
+    secondary_source_function : callable
+        A function that can be used to create the sound field of a
+        single secondary source.  See `sfs.mono.synthesize()`.
+
+    Notes
+    -----
+    `point_25d()` derives 2.5D WFS from the 3D
+    Neumann-Rayleigh integral (i.e. the TU Delft approach), cf. e.g.
+    Eq. (3.10), (3.11) in :cite:`Start1997`.
+    Also cf. Eq. (2.137) in :cite:`Schultz2016`.
+
+    The theoretical link of `point_25d()` and `point_25d_legacy()` was
+    introduced as *unified WFS framework* in :cite:`Firtha2017`.
+
+    .. math::
```

Comment:
In the rendered docs, it kinda looks like this equation is mentioned here in reference to `Firtha2017`.

But it's really just the equation that's implemented in this function.

Same for the "legacy" version below.

Probably the order should be somehow changed?
Some additional text to introduce the equation?

Reply:
good point, I've improved readability.